src/model.rs

```diff
@@ -73,6 +73,18 @@ pub struct StateDigest(pub Vec<u8>);
 #[derive(Debug, Serialize, Deserialize, Default, Clone)]
 pub struct BlobData(pub Vec<u8>);
 
+use crate::{
+    consensus::ConsensusCommand,
+    mempool::{MempoolCommand, MempoolResponse},
+};
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub enum BusMessage {
+    MempoolCommand(MempoolCommand),
+    MempoolResponse(MempoolResponse),
+    ConsensusCommand(ConsensusCommand),
+}
+
```

Comment:
Wonder if we shouldn't decentralise this and define the messages as newtypes in the relevant files instead, in the current architecture.